includes/Core/Notifications/Notifications.php

```diff
@@ -0,0 +1,262 @@
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Context $context Context instance.
+	 */
+	public function __construct( Context $context ) {
+		$this->context      = $context;
+		$this->google_proxy = new Google_Proxy( $this->context );
+		$this->credentials  = new Credentials( new Encrypted_Options( new Options( $context ) ) );
+	}
+
+	/**
+	 * Registers core notifications.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function register() {
+		add_filter(
+			'googlesitekit_rest_routes',
+			function ( $routes ) {
+				return array_merge( $routes, $this->get_rest_routes() );
+			}
+		);
+	}
+
+	/**
+	 * Gets related REST routes.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array List of REST_Route objects.
+	 */
+	private function get_rest_routes() {
+		$can_setup = function () {
+			return current_user_can( Permissions::SETUP );
+		};
+
+		return array(
+			new REST_Route(
+				'core/site/data/notifications',
+				array(
+					array(
+						'methods'             => WP_REST_Server::READABLE,
+						'callback'            => function ( WP_REST_Request $request ) {
+							$endpoint = add_query_arg(
+								array(
+									'site_id' => $this->credentials->get()['oauth2_client_id'],
+								),
+								$this->google_proxy->url( '/notifications' )
+							);
+							$response = wp_remote_get( $endpoint ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get
+
+							if ( is_wp_error( $response ) ) {
+								return $response;
+							}
+
+							try {
+								$response = $this->parse_response( $response );
+							} catch ( Exception $e ) {
+								return new WP_Error( 'exception', $e->getMessage() );
+							}
+
+							$data = array_map(
+								function ( Notification $notification ) {
+									return $notification->prepare_for_js();
+								},
+								$this->map_response_to_notifications( $response )
+							);
+
+							return new WP_REST_Response( $data );
+						},
+						'permission_callback' => $can_setup,
+					),
+				)
+			),
+			new REST_Route(
+				'core/site/data/mark-notification',
+				array(
+					array(
+						'methods'             => WP_REST_Server::EDITABLE,
+						'callback'            => function ( WP_REST_Request $request ) {
+							$data = $request['data'];
+
+							if ( empty( $data['notificationID'] ) ) {
+								return $this->missing_required_param( 'data.notificationID' );
+							}
+							if ( empty( $data['notificationState'] ) ) {
+								return $this->missing_required_param( 'data.notificationState' );
+							}
+
+							$credentials = $this->credentials->get();
+							$response    = wp_remote_post(
+								$this->google_proxy->url( '/notifications/mark' ),
+								array(
+									'body' => array(
+										'site_id'         => $credentials['oauth2_client_id'],
+										'site_secret'     => $credentials['oauth2_client_secret'],
+										'notification_id' => $data['notificationID'],
+										'notification_state' => $data['notificationState'],
```

Comment:
@felixarntz I'm not sure why, but PHPCS says that this alignment is correct.

Reply:
I think it wants to `=>` to be aligned?

```php
'site_id'            => $credentials['oauth2_client_id'],
'site_secret'        => $credentials['oauth2_client_secret'],
'notification_id'    => $data['notificationID'],
'notification_state' => $data['notificationState'],
```

Reply:
It should want that, but it automatically fixes it as highlighted above (which should be wrong)